jsonpickle/unpickler.py

```diff
@@ -555,7 +581,7 @@ def _restore_repr(self, obj):
         if self.safe:
             # eval() is not allowed in safe mode
             return None
-        obj = loadrepr(obj[tags.REPR])
+        obj = loadmodule(obj[tags.REPR])
```

Comment:
While what follows isn't a change for now, it would be nice eventually to rename tags.REPR to tags.MODULE or something like that, and provide a helper for decoding tags.REPR strings. This could be done before 4.0.0, as long as the helper function is invoked automatically when an instance of tags.REPR is detected. It could also smooth the transition to a new tags.MODULE if it saves written/updated using the new tags.MODULE.